### ISSUE TITLE
Staging Review finding: Implicit submit results in change of context

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#128357]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Appointments Team
**Product:** Appointments
**Feature:** Oracle Health Scheduling - Direct Schedule and Request for an Appointment

## Findings details
**VA.gov Experience Standard - issue:** [User encounters design components or patterns that are inconsistent or confusing.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

Sort providers contains an implicit submit, which can be confusing and interrupt asssitive technology users. 

<details><summary>Link, screenshot or steps to recreate</summary>

https://github.com/user-attachments/assets/0e6b2607-70b6-4863-a194-4f3ae8fd28b7

</details>

## Recommended action

Avoid implicit submits whenever possible, as this can initiate a change in context, which is a challenge for assistive technology users. 
https://www.w3.org/WAI/WCAG21/Techniques/html/H84

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 3.2.2 On Input - Level A](https://www.w3.org/TR/WCAG22/#on-input)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by outerpress** [comment]
@sarahhortonadhoc another one related to the sort and filter pattern

**Comment by outerpress** [comment]
@jasonday @sarahhortonadhoc just for some more background, we're waiting for this pattern to be developed and tested by another team before we implement it. I added it to #97264, which holds the 2 other requests from collab cycle to implement this functionality. I think [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/128669) is the most likely to get us there, so we'll track that, and look at picking up the work on our end when that's done and the pattern is in VADS.

**Comment by outerpress** [comment]
@sarahhortonadhoc done! https://github.com/department-of-veterans-affairs/va.gov-team/issues/133972
